### DISCUSSION TITLE
Why does building on windows hosted agents takes 15min when it takes about 1min on my machine?

### DISCUSSION THREAD

**Discussion by FrancoisM** [actions/virtual-environments#5282] [General questions]
I have no clue on why our builds take ages (15min) on windows hosted agents whereas building it on our local machines takes about 1min.

Would anyone know?

**Comment by miketimofeev** [comment]
@FrancoisM GitHub hosted runners have 2 cores and 7Gb or Ram — that may explain the difference
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
